src/fdrs_gateway_espnow.h

```diff
@@ -470,15 +472,19 @@ void sendESPNow(uint8_t address)
 
 void recvTimeEspNow(uint32_t t) {
   // Process time if there is no master set yet or if LoRa is the master or if we are already the time master
-  if(timeMaster.tmType == TM_NONE || timeMaster.tmType == TM_LORA || (timeMaster.tmType == TM_ESPNOW && timeMaster.tmAddress == incMAC[4] << 8 | incMAC[5])) {
+  if(timeMaster.tmNetIf <= TMIF_ESPNOW ) {
     DBG("Received time via ESP-NOW from 0x" + String(incMAC[5], HEX));
-    if(timeMaster.tmAddress == 0x0000) {
-      timeMaster.tmType = TM_ESPNOW;
-      timeMaster.tmAddress = incMAC[4] << 8 & incMAC[5];
-      DBG("ESP-NOW time master is 0x" + String(incMAC[5], HEX));
+    if(timeMaster.tmNetIf < TMIF_ESPNOW) {
+      timeMaster.tmNetIf = TMIF_ESPNOW;
+      timeMaster.tmAddress = incMAC[4] << 8 | incMAC[5];
```

Comment:
I'm not great with bitwise operations yet, could you save me some research and tell me what `incMAC[4] << 8 | incMAC[5]` is expressing in English?

In the following, I understand the second condition is that the address is not 0x00, but what is the fist check for?
```if((timeMaster.tmAddress != (ESPNOW1[4] << 8 | ESPNOW1[5])) && ESPNOW1[5] != 0x00) {```

Thanks, sorry for the basic question.